src/test/java/subway/integration/LineIntegrationTest.java

```diff
@@ -72,70 +75,144 @@ void createLineWithDuplicateName() {
         assertThat(response.statusCode()).isEqualTo(HttpStatus.BAD_REQUEST.value());
     }
 
-    @DisplayName("지하철 노선 목록을 조회한다.")
+    @DisplayName("지하철 모든 노선을 조회한다.")
     @Test
     void getLines() {
         // given
-        ExtractableResponse<Response> createResponse1 = RestAssured
+        ExtractableResponse<Response> lineResponse1 = RestAssured
                 .given().log().all()
                 .contentType(MediaType.APPLICATION_JSON_VALUE)
                 .body(lineRequest1)
                 .when().post("/lines")
-                .then().log().all().
-                extract();
+                .then().extract();
 
-        ExtractableResponse<Response> createResponse2 = RestAssured
+        ExtractableResponse<Response> lineResponse2 = RestAssured
                 .given().log().all()
                 .contentType(MediaType.APPLICATION_JSON_VALUE)
                 .body(lineRequest2)
                 .when().post("/lines")
-                .then().log().all().
-                extract();
+                .then().extract();
 
-        // when
-        ExtractableResponse<Response> response = RestAssured
+        Map<String, String> stationParams1 = new HashMap<>();
+        stationParams1.put("name", "해운대역");
```

Comment:
```suggestion
        Map<String, String> stationParams1 = new HashMap<>();
        stationParams1.put("name", "해운대역");
```
Map 타입 보단 실제 요청객체 타입으로 생성합시당!

Reply:
아무 생각없이 구현했었네요 😅
Request 객체 생성하는 방식으로 수정했습니다!